### ISSUE TITLE
Add other way to make a donation

### ISSUE THREAD

**Issue by rodislav** [mplushnikov/lombok-intellij-plugin#608]
Hello, 

I would like to make a donation but PayPal is not an option today. Is it possible for you to extend the way people can donate to your project ? 
Patreon, Amazon vouchers, etc.

Thanks

**Comment by mplushnikov** [comment]
Hello @rodislav,
thank you for your wish to support the plugin project!
May I ask, why Paypal ist not a option for you? I want to  understand better your troubles with it to retink any other solutions.

**Comment by rodislav** [comment]
Hello @mplushnikov, 
when I opened the ticket I tried to make a donation with a credit card and for some reason it was possible to use only a paypal account, now I tried again to make sure the case is reproducible and I had an option to make a donation just with a credit card, even make it monthly one.

I can say that the issue can be close now.
I'm sorry for late answer.

**Comment by rodislav** [comment]
Hm, I tried to use just a credit card and I was asked to create a paypal account :-( 
so I still cannot do it.

**Comment by mplushnikov** [comment]
@rodislav Thank you Rodislav. I have to investigate wich other options I can add. Paypal was the simplest one to add. I looked at Patreon, but it looks like to needs a lot of configuration to start with. Do you have some other easy to create suggestions?

**Comment by jcputney** [comment]
@mplushnikov Stripe could be a good option https://donorbox.org/?utm_source=Stripe

**Comment by rodislav** [comment]
Indeed, stripe seems to be nice, you can check how oppencollective works, yet I'm not sure if it'll fit your case.

**Comment by vadipp** [comment]
I wanted to donate $2 for a start, but Paypal wants another $2 in fees. That's just not right. Please add a way to donate with more sensible fees or no fees at all (this is open-source, after all). Thank you for your work! :)
